gatsby-node.ts

```diff
@@ -1,9 +1,12 @@
 import { createFileNodeFromBuffer } from 'gatsby-source-filesystem';
+import fetch from 'node-fetch';
```

Comment:
I thought about getting rid of this dependency and let use Gatsby the Fetch API from [nodejs](https://nodejs.org/en/blog/announcements/v18-release-announce/)

Reply:
That worked I've tried it a few days ago. What do you think?

Reply:
I run into the following error when I remove the import:
`ExperimentalWarning: The Fetch API is an experimental feature. This feature could change at any time`

Reply:
Yes, but that's only a warning 😅

Reply:
OK